packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -196,20 +196,10 @@ export class WallectConnectSession {
 	private async waitForConfirmation(
 		txId: string
 	): Promise<algosdk.modelsv2.PendingTransactionResponse> {
-		const response = await this.algodClient.status().do();
-		let lastround = response[LAST_ROUND];
-		/*eslint no-constant-condition: ["error", { "checkLoops": false }]*/
-		while (true) {
-			const pendingInfo = await this.algodClient.pendingTransactionInformation(txId).do();
-			if (pendingInfo["pool-error"]) {
-				throw new Error(`Transaction Pool Error: ${pendingInfo["pool-error"] as string}`);
-			}
-			if (pendingInfo[CONFIRMED_ROUND] !== null && pendingInfo[CONFIRMED_ROUND] > 0) {
-				return pendingInfo as algosdk.modelsv2.PendingTransactionResponse;
-			}
-			lastround++;
-			await this.algodClient.statusAfterBlock(lastround).do();
-		}
+		const pendingInfo = await algosdk.waitForConfirmation(this.algodClient, txId, 10);
```

Comment:
Shall we store `10`(waitRounds) as constant and use it in all web modes(my wallet, wallet connect, etc,...)?

Reply:
yeah, that's a nice suggestion. Will do